src/Configuration.php

```diff
@@ -1154,6 +1154,8 @@ public function getOutputDecorated()
             case self::COLOR_MODE_DISABLED:
                 return false;
         }
+
+        return null;
```

Comment:
`colorMode()` always returns a value, and it's always one of the above; the switch narrows the possible types down to "never". i get that this might not be the easiest thing to statically analyze, but in the event that the return type for `colorMode` changed, this would give the wrong answer anyway :) 

I think the right fix is to move the `COLOR_MODE_AUTO` case to the bottom and add a `default`. Is that sufficient to get PHPStan to stop complaining?

Reply:
Yes, it would be enough 👍